scripts/install/ReplaceMetadataExtractor.php

```diff
@@ -1,48 +0,0 @@
-<?php
```

Comment:
The 2 removed classes exist since April, are we sure this is safe to remove. 

Summoning @bartlomiejmarszal since he introduced this. 

We need to make sure this is not creating a breaking change that can affect next exports/imports

Reply:
Yes, we will discuss it with Bart as we didnt find any reasons to have this extractor